### PR TITLE
Add new integration for AiDot

### PR DESCRIPTION
**Pull request by s1eedz**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/117337
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5485
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/aidot.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ha_codeowners:
+  - '@sz411'
+ha_domain: Aidot
+ha_homekit: true
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+The [Aidot](https://www.aidot.com/) integration allows you to control the Wi-Fi lights with home assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/aidot.markdown]:
```suggestion
The [Aidot](https://www.aidot.com/) {% term integration %} allows you to control the Wi-Fi lights with Home Assistant.
```

File: source/_integrations/aidot.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+{% include integrations/config_flow.md %}
+
+## Supported Devices
```
Comment (c0ffeeca7) [inline source/_integrations/aidot.markdown]:
```suggestion
## Supported devices
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headers.

File: source/_integrations/aidot.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Aidot
+description: Instructions on integrating Aidot home security with Home Assistant.
+ha_category:
+  - Light
+ha_release: 1.0.0
```
Comment (c0ffeeca7) [inline source/_integrations/aidot.markdown]: This will need to be updated to the release when it is merged. 
```suggestion
ha_release: 2024.6.
```

**Comment by s1eedz** [comment]
Add PR link for home assistant docs ： https://github.com/home-assistant/home-assistant.io/pull/32704

Add PR link for home assistant brands：https://github.com/home-assistant/brands/pull/5438

File: source/_integrations/aidot.markdown [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Aidot
+description: Instructions on integrating Aidot home security with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.6
+ha_iot_class: local_polling
+ha_config_flow: true
+ha_codeowners:
+  - '@sz411'
```
Comment (c0ffeeca7) [inline source/_integrations/aidot.markdown]:
```suggestion
  - '@s1eedz'
```
Was sz411 listed here on purpose? The core PR has your name on it. If you are the code owner, your name should be here.

**Pull request closed without merging**